.github/workflows/build.yml

```diff
@@ -238,7 +238,7 @@ jobs:
     timeout-minutes: 15
     runs-on: ubuntu-latest
     # Size Check will error out outside of the context of a PR
-    if: ${{ github.event_name == 'pull_request' }} || ${{ startsWith(github.ref, 'refs/heads/master/') }}
+    if: github.event_name == 'pull_request' || startsWith(github.ref, 'refs/heads/master/')
```

Comment:
Note: This previously always evaluated to `true`. No need to wrap the single expressions in `${{ }}`, as the whole `if` expression is auto-wrapped with it.